tests/test_agents/test_counter.py

```diff
@@ -56,6 +56,8 @@ def _send_tx_to_node(
         """
         # at least two hex digits
         tx_arg = "0x{:02x}".format(value)
+        log_message = "requesting broadcast_tx_commit"
+        logging.debug(f"{log_message}: {node_address} {tx_arg}")
         result = requests.get(
             node_address + "/broadcast_tx_commit",
             params=dict(tx=tx_arg),
```

Comment:
This appears to be the issue for Counter

Reply:
no more was an issue with image building now resolved (cause: `int64` to `uint64`)